packages/apm/src/integrations/tracing.ts

```diff
@@ -283,6 +283,52 @@ export class Tracing implements Integration {
     });
   }
 
+  /**
+   * Returns a new Transaction either continued from sentry-trace meta or a new one
+   */
+  private static _getNewTransaction(hub: Hub, transactionContext: TransactionContext): Transaction {
+    let traceId;
+    let parentSpanId;
+    let sampled;
+
+    const header = Tracing._getMeta('sentry-trace');
+    if (header) {
+      const span = SpanClass.fromTraceparent(header);
+      if (span) {
+        traceId = span.traceId;
+        parentSpanId = span.parentSpanId;
+        sampled = span.sampled;
+      }
+      Tracing._log(
+        `[Tracing] found 'sentry-meta' '<meta />' continuing trace with: trace_id: ${traceId} span_id: ${parentSpanId}`,
+      );
+    }
+
+    return hub.startTransaction({
+      parentSpanId,
+      sampled,
+      traceId,
+      trimEnd: true,
+      ...transactionContext,
+    }) as Transaction;
+  }
+
+  /**
+   * Returns the value of a meta tag
+   */
+  private static _getMeta(metaName: string): string | null {
+    const metas = document.getElementsByTagName('meta');
+
+    // tslint:disable-next-line: prefer-for-of
+    for (let i = 0; i < metas.length; i++) {
+      if (metas[i].getAttribute('name') === metaName) {
+        return metas[i].getAttribute('content');
+      }
+    }
+
+    return null;
+  }
```

Comment:
```suggestion
  private static _getMeta(metaName: string): string | undefined {
    return (document.querySelector(`meta[name=${metaName}]`) || {}).content;
  }
```

Reply:
This has typescript errors, I will go with 
```
const el = document.querySelector(`meta[name=${metaName}]`);
return el ? el.getAttribute('content') : null;
```

Reply:
Ah, silly empty object typing 🎳